Resources/Prototypes/_LostParadise/Maps/Pools/LostParadiseMapPool.yml

```
@@ -1,20 +1,20 @@
 - type: gameMapPool
   id: LostParadiseMapPool
   maps:
-#  - LPPAsterisk
-#  - LPPSubmarine
-#  - LPPPebble
+  - LPPAsterisk
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)